style/activity.css

```diff
@@ -42,13 +42,38 @@
   top: 0px;
   background: var(--jp-layout-color0);
   z-index: 2;
+  display: flex;
+  justify-content: space-between;
 }
 
 .Verdant-events-date-header-label {
   margin-left: 1.5em;
   margin-top: 2px;
 }
 
+.Verdant-events-date-header-arrow {
+  display: inline-block;
+  vertical-align: text-bottom;
+  background-image: var(--verdant-icon-datearrow-open);
+  width: 1.4em;
+  height: 1.4em;
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: 1.4em;
+  padding: 5px;
+}
+
+.Verdant-events-date-header-arrow.closed {
+  background-image: var(--verdant-icon-datearrow-closed);
+  width: .8em;
+  height: .8em;
+  background-size: 1em;
```

Comment:
Should be able to delete these rules entirely if you can safely reduce the base class .Verdant-events-date-header-arrow to just the width sizing rule. Again though, check that this looks OK in your browser, since browser displays can differ.